mne/io/brainvision/brainvision.py

```diff
@@ -204,13 +205,15 @@ def _read_vmrk(fname):
     # extract event information
     items = re.findall(r"^Mk\d+=(.*)", mk_txt, re.MULTILINE)
     onset, duration, description = list(), list(), list()
-    date_str = None
+    date_str = ''
```

Comment:
`None` has a logic. It's not the same reading empty string that not reading anything. Do we want to keep that logic or not? I'm not sure.

Reply:
We could keep it, but then we need to build `None` handling into `_str_to_meas_date`, I prefer the empty string option, which is also documented in the docstring of this function (see line 155)